### PR TITLE
Altered old migration

### PR DESCRIPTION
**Pull request by He7i**
copy uuid into temp column
then delete old column
rename temp column

### CODE REVIEW

**Comment by He7i** [comment]
> I have a few questions that I need answers to before we roll out a change on the specific migration:
> 
>     1. Did you get the migrations to run on infraqa, and does this pull request rather just fix that?
> 
>     2. Is the reason that the migration didn't go out due to an error in the SQL and therefore the migration not entering itself into the _migration_version_ table?
> 
>     3. This might be related to question #2, but can you confirm that the migrations have not run on any other QA machines - because if the migration has run on any other machine, then your changes will definitely not get to the machine

#1:In the migrations log it showed my previous script had an error so this is supposed to fix that
#2:The migration didnt go into the migration table because my migration didnt run because it had an error
#3:This is can not do

File: migrations/main/2021-01-14-alterations-for-spatial-linkrer.sql [hunk 1]
```diff
@@ -1,7 +1,11 @@
 ALTER TABLE "LinkingMainAndMirror" ADD COLUMN "left_table_name" VARCHAR;
 ALTER TABLE "LinkingMainAndMirror" ADD COLUMN "left_dbname" VARCHAR;
 ALTER TABLE "LinkingMainAndMirror" ADD COLUMN "left_id_column" VARCHAR;
-ALTER TABLE "LinkingMainAndMirror" RENAME "main_id" TO "left_id";
-ALTER TABLE "LinkingMainAndMirror" ALTER COLUMN "left_id" TYPE VARCHAR;
 ALTER TABLE "LinkingMainAndMirror" DROP COLUMN "main_table_name";
 ALTER TABLE "LinkingMainAndMirrorTrackTable" ADD COLUMN "left_dbname" VARCHAR;
+ALTER TABLE "LinkingMainAndMirror" ADD COLUMN "temporary_id" VARCHAR;
```
Comment (FritzOnFire) [inline migrations/main/2021-01-14-alterations-for-spatial-linkrer.sql:6]: Why call it `temporary_id`? Why not just call it `left_id` from the start?

File: schema/main.schema [hunk 2]
```diff
@@ -9072,8 +9072,9 @@ CREATE TABLE "LinkingMainAndMirror" LinkingMainAndMirror
 	optional	text		left_table_name					""
 	optional	text		left_dbname						""
 	optional	text		left_id_column					""
+	optional	text		temporary_id					""
```
Comment (FritzOnFire) [inline schema/main.schema:9075]: Why is this here?

File: schema/main.schema [hunk 3]
```diff
@@ -9072,8 +9072,9 @@ CREATE TABLE "LinkingMainAndMirror" LinkingMainAndMirror
...
 "1.As_built Data"
-	optional	text		left_id							"ID In Engineering Table"			tags:allPropFromMainTable
+	optional	uuid		main_id							"ID In Engineering Table"			tags:allPropFromMainTable
```
Comment (FritzOnFire) [inline schema/main.schema:9077]: This does not look right

**Review by FritzOnFire — CHANGES_REQUESTED** [review]
Now, its still not running on my db, which leads me to believe that you have most likely never ran all of the services on your machine. I would recommend that you figure out how to get the schema of the main db from infraqa and then create a new db on your side and run/restore that schema there and test this migration with that new db.
